config/application.rb

```diff
@@ -13,7 +13,7 @@ module Nucore
 
   class Application < Rails::Application
 
-    config.load_defaults 6.0
+    config.load_defaults 6.1
```

Comment:
Have you gone through all of the rails 6.1 defaults to ensure that they work well, for example:

```
# Specify cookies SameSite protection level: either :none, :lax, or :strict.
#
# This change is not backwards compatible with earlier Rails versions.
# It's best enabled when your entire app is migrated and stable on 6.1.
# Rails.application.config.action_dispatch.cookies_same_site_protection = :lax

# Generate CSRF tokens that are encoded in URL-safe Base64.
#
# This change is not backwards compatible with earlier Rails versions.
# It's best enabled when your entire app is migrated and stable on 6.1.
# Rails.application.config.action_controller.urlsafe_csrf_tokens = true
```

Reply:
@rdunlop I think these are ok to test in production - using [this article](https://lilyreile.medium.com/rails-6-1-new-framework-defaults-what-they-do-and-how-to-safely-uncomment-them-c546b70f0c5e) as a reference.